controllers/users.controller.js

```diff
@@ -0,0 +1,12 @@
+const {selectUsers} = require('../models/users.model');
+
+async function getUsers(req, res, next) {
+    try {
+        const users = await selectUsers();
+        res.status(200).send({users});
+    } catch (error) {
+        
```

Comment:
don't forget to pass any errors from your controller to your error handling middleware using next! this isn't doing anything currently